website/content/en/docs/kubebuilder/references/client.md

```diff
@@ -431,21 +423,21 @@ func (r *ReconcileApp) Reconcile(request reconcile.Request) (reconcile.Result, e
 
 ```Go
 import (
-	"context"
-	"reflect"
-
-	appv1alpha1 "github.com/example-org/app-operator/pkg/apis/app/v1alpha1"
-
-	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+    "context"
+    "reflect"
+
+    appv1alpha1 "github.com/example-org/app-operator/pkg/apis/app/v1alpha1"
```

Comment:
Look for all paths that are of the form `pkg/apis/app/v1alpha1` and update them to the new layout paths `api/v1alpha1`.
Similar to what we did for the controller paths.
```suggestion
    appv1alpha1 "github.com/example-org/app-operator/api/v1alpha1"
```